chuzksy-codeactive/eventcity-react
# #159032001: Implement e2e test

#### What does this PR do?
- Implement e2e test for the users
- Implement e2e test for centers
- Implement e2e test for events
#### Description of Task to be completed?
Implement end-to-end test
#### How should this be manually tested?
Before testing make sure Java JDK is installed and install `nightwatch` globally
- clone the repo and cd into the repo directory
- on the terminal type, `npm install` then `npm run e2e:test` to run `nightwatch`
#### Any background context you want to provide?
No
#### What are the relevant pivotal tracker stories?
#159032001
